.github/workflows/ci.yml

```diff
@@ -63,24 +72,19 @@ jobs:
           # https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions
           is_dependabot="${{ github.actor == 'dependabot[bot]' }}"
 
-          # Are we running on a push event
-          # Many jobs/steps only execute or change execution on push events
-          is_push="${{ github.event_name == 'push' }}"
 
           if [[ "${{ github.event_name }}" == 'pull_request' ]]; then
             # repository on a pull request refers to the base which is always mozilla/addons-server
             is_fork=${{ github.event.pull_request.head.repo.fork }}
           else
-            # Repository on a push refers to the head which would be the fork
+            # In most events, the epository refers to the head which would be the fork
```

Comment:
the else statement applies to `workflow_dispatch`, `push` and `release` all of which share the same logic